astropy/coordinates/representation.py

```diff
@@ -718,9 +977,32 @@ def get_xyz(self, xyz_axis=0):
 
     @classmethod
     def from_cartesian(cls, other):
+        """
```

Comment:
Since we're using `InheritDocStrings`, maybe just put the docstring on the very base?

Note that we're not converting here, so if we add a specific docstring to `Cartesian` it should say so.